en/docs/chapter_backtracking/backtracking_algorithm.md

```diff
@@ -1,42 +1,42 @@
 # Backtracking algorithms
 
-<u>Backtracking algorithm</u> is a method to solve problems by exhaustive search, where the core idea is to start from an initial state and brute force all possible solutions, recording the correct ones until a solution is found or all possible choices are exhausted without finding a solution.
+<u>Backtracking algorithm</u> is a method to solve problems by exhaustive search. Its core concept is to start from an initial state and brute force all possible solutions. The algorithm records the correct ones until a solution is found or all possible choices are exhausted without success.
```

Comment:
brute force all possible solutions => brutally search for all possible solutions

Reply:
all possible choices are exhausted without success => all possible solutions have been tried but no solution can be found.